config/config.defaults.yml

```diff
@@ -440,16 +440,10 @@ default: &default_settings
   # Stripe allows to hold funds up to 90 days
   stripe_max_booking_date: 90
 
-  # :platform - marketplace API account is responsible for fees, refunds, etc. funds are captured to platform account and then moved to seller by Transfer
-  # :seller   - Custom Account for seller is responsible for fees, funds are sent by Payout
-  stripe_charge_destination: :platform
-
-  # Disable displaying stored cards for customer
-  stripe_hide_stored_cards: true
-
-  # :connect - use only Stripe Connect
-  # :managed - use only Managed
-  stripe_accounts_mode: :managed
+  # :separate    - Separate charges and transfers 
+  # :direct      - Direct charges
+  # :destination - Destination charges
+  stripe_charges_mode: :destination
```

Comment:
I think it's useful to have a short description of each type of charging model here. Or at least a link to the Stripe documentation: https://stripe.com/docs/connect/charges